bmu/Src/batteries.c

```diff
@@ -1572,7 +1574,7 @@ void batteryTask(void *pvParameter)
  */
 
 /// The period to send cell voltage and temperature CAN messages
-#define CAN_CELL_SEND_PERIOD_MS 50
+#define CAN_CELL_SEND_PERIOD_MS 53
```

Comment:
Why is this needed? Should use a round number

Reply:
I was trying to use prime numbers so that the times don’t line up for when the can messages are spammed